code/API_definitions/region-device-count.yaml

```
@@ -62,7 +62,7 @@ externalDocs:
   description: Project documentation at CAMARA
   url: https://github.com/camaraproject/RegionDeviceCount
 servers:
-  - url: '{apiRoot}/regionDeviceCount/vwip'
+  - url: '{apiRoot}/regionDeviceCount/r1.1'
```

The URL is derived from the API Version, not the release number, cf https://github.com/camaraproject/Commonalities/blob/r2.1/documentation/API-design-guidelines.md#53-api-versions-throughout-the-release-process

```suggestion
  - url: '{apiRoot}/regionDeviceCount/v0.1rc1'
```

---

URL has been changed in https://github.com/camaraproject/RegionDeviceCount/pull/47/commits/dcdf595faa2fbffa52e24baead788ae5a2491d6e